### PR TITLE
Adding JUnit. Fixes #3874

### LINKED ISSUE #3874: test: improve coverage
The code coverage https://codecov.io/github/ctripcorp/apollo seems to be less than 60%.

It would be great if it can be brought up around 80-90%.

### PR DESCRIPTION
**Pull request by ayush0407**
## What's the purpose of this PR

This PR is for JUnit fix and increased coverage and logic test.

XXXXX

## Which issue(s) this PR fixes:
Fixes #3874

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [ ] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [ ] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [ ] Update the [`CHANGES` log](https://github.com/ctripcorp/apollo/blob/master/CHANGES.md).

### CODE REVIEW

**Review by nobodyiam — COMMENTED** [review]
Thanks for the contribution, would you please follow the [code style](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md#code-conventions) to format these 2 files?

**Comment by ayush0407** [comment]
> Thanks for the contribution, would you please follow the [code style](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md#code-conventions) to format these 2 files?
> 
> BTW, the new file need to have the copyright comment in its header.

Sure, I'll fix that and check in the code.

**Comment by ayush0407** [comment]
Hi there, I added the copyright statement in the new file but can you please help me with the code format. I didn't get that part?

**Comment by nobodyiam** [comment]
@ayush0407 

According to the [Code Conventions](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md#code-conventions), you need to import the code style files into your IDE, and use that to format your code changes.
If you don't use any IDE, then you may manually format the code changes, e.g. indent size is 2, tab size is 2, use space to replace tab, etc.

image

File: apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.beans.factory.support.BeanDefinitionRegistry;
+import org.springframework.beans.factory.support.SimpleBeanDefinitionRegistry;
+
+/**
+ * @author Ayush Jha
```
Comment (zouyx) [inline apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java:36]: No author in the new file.Remove it pls.

Comment (ayush0407) [reply apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java:36]: So I don't have to add the author name in the new file right?

Comment (ayush0407) [reply apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java]: I just removed the author and reformatted the file. Please verify it and do let me know.

**Review by zouyx — APPROVED** [review]

**Review by zouyx — COMMENTED** [review]
Fix test case pls

File: apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java [hunk 2]
```diff
@@ -17,49 +17,68 @@
...
 
-  private final ConfigChangeContentBuilder configChangeContentBuilder = new ConfigChangeContentBuilder();
-  private String configString;
-
-  @Before
-  public void initConfig() {
-
-    Item createdItem = MockBeanFactory.mockItem(1, 1, "timeout", "100", 1);
-    Item updatedItem = MockBeanFactory.mockItem(1, 1, "timeout", "1001", 1);
+	private final ConfigChangeContentBuilder configChangeContentBuilder = new ConfigChangeContentBuilder();
```
Comment (nobodyiam) [inline apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java:35]: The code format for this file is not right, please re-format it.

**Comment by ayush0407** [comment]
> Fix test case pls

Which one specifically? Is any of the test breaking?

File: apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java (hunk 2 above)
Comment (ayush0407) [reply apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java:35]: sure

Comment (ayush0407) [reply apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java]: How can I fix that? I set the format style to the one you provided. Please tell me what's wrong here.

Comment (nobodyiam) [reply apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java]: After you set the format style, you need to use the ide's built-in `Reformat Code` function to format the changed lines. Then the `tab` characters would be replaced with `space` characters.

```java
	private ConfigChangeContentBuilder configChangeContentBuilder;
```
```java
  private ConfigChangeContentBuilder configChangeContentBuilder;
```

Comment (ayush0407) [reply apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java]: Yes, I did that now. Please do check it.

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**